public/scripts/app.js

```diff
@@ -1,7 +1,85 @@
 console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
+  console.log("Document has loaded!")
 
-// your code
 
-});
+  let successPostAlert = `<div class="alert alert-success" role="alert">Contact successfully added!</div>`
+  let errorAlert = `<div class="alert alert-danger" role="alert">Oops! Something went wrong!</div>`
+
+  $.ajax({
+    method: 'GET',
+    url: '/api/contacts',
+    success: onSuccess,
+    error: onError
+  });
+
+  $('#addContact').on('click', function(e) {
+    $('#contactModal').modal(); //triggers modal
+    console.log("modal open!")
+      $('form').on('submit', function(e) {
+        console.log("button clicked");
```

Comment:
Since you don't `e.preventDefault()` here, you refresh the page, even though you're making an AJAX call.